ops/build_and_package.sh

```diff
@@ -35,12 +35,18 @@ then
   exit 1
 fi
 
+if [ ! -f ../bcda/models/fhir/alr/hcc_crosswalk.csv ]
+then
+  echo "Crosswalk flie must exist prior to creating package."
+  exit 1
+fi
+
 cd ../bcda
 go clean
 echo "Building bcda binary..." 
 go build -ldflags "-X github.com/CMSgov/bcda-app/bcda/constants.Version=$VERSION"
 echo "Packaging bcda binary into RPM..."
-fpm -v $VERSION -s dir -t rpm -n bcda bcda=/usr/local/bin/bcda swaggerui=/etc/sv/api
+fpm -v $VERSION -s dir -t rpm -n bcda bcda=/usr/local/bin/bcda swaggerui=/etc/sv/api models/fhir/alr/hcc_crosswalk.csv=/etc/sv/api/hcc_crosswalk.csv
```

Comment:
I wonder if we should have the conf package handle the crosswalk.

Reply:
Let's chat about this when you're online. I was wondering if this falls in as an application config or application data. Thinking more it as data that is stored as a file. But definitely open to see if it makes more sense in conf package/stored elsewhere.